third_party/nvfuser/csrc/lower_index.cpp

```diff
@@ -1216,7 +1216,8 @@ void IndexLowering::handleGroupedGridWelford(
 
 void IndexLowering::handle(const LoadStoreOp* ldst) {
   const auto in = lowerSrcIndex(ldst->in(), ldst->out(), {}, true);
-  const auto out = lowerDstIndex(ldst->out(), {}, true);
+  const auto out =
```

Comment:
Could you please remind here about `LoadStoreOp` and when `ir_utils::isLdMatrixOp(ldst)` is `false`?

Reply:
Currently, `LoadStoreOp` can only be `ld.matrix` and `cp.async`. In the future, when we start adding Hopper support, it can also be TMA load and store. So today: `!ldmatrix` means `is cpasync`.

Reply:
Oh, I see. Please also add that as  a code comment.

I was wondering if vectorized loads and stores are also `LoadStoreOp`, which I remember we discussed before but didn't remember if we did anything with them.

Reply:
added